improver_tests/utilities/cube_manipulation/test_filter_realizations.py

```diff
@@ -59,7 +59,12 @@ def realization_cubes_fixture() -> CubeList:
             )
         )
     cube = cubes.merge_cube()
-    return CubeList(cube.slices_over("realization"))
+    sliced_cubes = CubeList(cube.slices_over("realization"))
+    [
+        s.attributes.update({"history": f"20171110T{i:02d}00Z"})
+        for i, s in enumerate(sliced_cubes)
+    ]
```

Comment:
This is just adding a history attribute that is different for each realization, to force the `filter_realizations` method to remove it again.